### PR TITLE
Introduce a build configuration without accessibility services

### LINKED ISSUE #4624: [FEATURE] Remove Accessibility API usage from Play Store Builds
### Describe the feature

Google has not approved our usage of the accessibility API and we need to remove it.

### Why would it be useful to add?

It is the only remaining Play Store re-release.

### Did the feature exist in Lawnchair v2? (Play Store version)

No

### Example(s)

Not needed.

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by yasanglass**
## Description

Required for Play Store release.

Fixes #4624

## Type of change

:x: General change (non-breaking change that doesn't fit the below categories like copyediting)
:x: Bug fix (non-breaking change which fixes an issue)
:white_check_mark: New feature (non-breaking change which adds functionality)
:x: Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

**Review by MrSluffy — CHANGES_REQUESTED** [review]
A̶l̶l̶ l̶o̶o̶k̶s̶ g̶o̶o̶d̶ t̶o̶ m̶e̶, j̶u̶s̶t̶ a̶d̶d̶ t̶h̶e̶ n̶e̶w̶ f̶l̶a̶v̶o̶r̶ i̶n̶ c̶i̶.y̶m̶l̶ T̶h̶a̶n̶k̶s̶!̶

File: build.gradle [hunk 1]
```diff
@@ -234,7 +234,7 @@ android {
     }
 
     // See: https://developer.android.com/studio/build/build-variants#flavor-dimensions
-    flavorDimensions += ["app", "recents", "channel"]
+    flavorDimensions += ["app", "recents", "channel", "accessibility"]
```
Comment (Goooler) [inline build.gradle]: You don't have to add a new flavor, just place accessibility related code under `github` falvor.

Comment (MrSluffy) [reply build.gradle]: agree since the market is for playstore all along

Comment (yasanglass) [reply build.gradle]: Good point- I forgot that we do not release on any other place other than these two...

Comment (yasanglass) [reply build.gradle]: Should be OK now

File: quickstep/AndroidManifest-launcher.xml [hunk 2]
```diff
@@ -25,7 +25,7 @@
 
     <uses-sdk android:targetSdkVersion="33" android:minSdkVersion="26"
         tools:overrideLibrary="com.kieronquinn.app.smartspacer.sdk.client"/>
-    
+
```
Comment (Goooler) [inline quickstep/AndroidManifest-launcher.xml]:
```suggestion
    
```

**Review by Goooler — APPROVED** [review]

**Review by MrSluffy — APPROVED** [review]

**Pull request merged**